apollo-federation/src/query_graph/mod.rs

```diff
@@ -389,10 +397,13 @@ pub struct QueryGraph {
     /// lowered composition validation on a big composition (100+ subgraphs) from ~4 minutes to
     /// ~10 seconds.
     non_trivial_followup_edges: IndexMap<EdgeIndex, Vec<EdgeIndex>>,
-    /// Maps each subgraph name to each field argument of `@fromContext`.
```

Comment:
Refactors:
- Similarly to above, there's renames here for clarity/consistency across QP data structures.
- It turns out `subgraph_to_args` isn't really used anywhere (there's a usage in JS QP, but it's replaceable by `arguments_to_context_ids_by_source`)